tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/registered_cope_survey_suppression_test.py

```diff
@@ -0,0 +1,134 @@
+                    -- COVID-19 or flu-like symptoms? --
+            -- 1333235 Which of the following symptoms did you have? (select all that apply). --
+            -- 1310065 Which of the following symptoms did you have? Select all that apply. Please specify: --
+            -- 1333012 What breathing treatment did you receive? Please select all that apply. --
+            -- 1333234 What other breathing treatment did you receive? Please specify. --
+            -- 1332769 Who do you know who has died? Check all that apply. --
+            -- 702686  Who do you know who has died? Please specify. --
+            -- 1333156 Do you think you have had COVID-19? --
+            -- 713888  When did your COVID-19 symptoms begin? --
+              (1, 101, 0, 0, 0, 1310058, 0, 0, 0, '2020-01-01'),
+              (2, 102, 0, 0, 0, 1333325, 0, 0, 0, '2020-01-01'),
+              (3, 103, 0, 0, 0, 1333235, 0, 0, 0, '2020-01-01'),
+              (4, 104, 0, 0, 0, 1310065, 0, 0, 0, '2020-01-01'),
+              (5, 105, 0, 0, 0, 1333012, 0, 0, 0, '2020-01-01'),
+              (6, 106, 0, 0, 0, 1333234, 0, 0, 0, '2020-01-01'),
+              (7, 107, 0, 0, 0, 1332769, 0, 0, 0, '2020-01-01'),
+              (8, 108, 0, 0, 0, 702686, 0, 0, 0, '2020-01-01'),
+              (9, 109, 0, 0, 0, 715714, 0, 0, 0, '2020-01-01'),
+              (10, 110, 0, 0, 0, 1333156, 0, 0, 0, '2020-01-01'),
+              (11, 111, 0, 0, 0, 713888, 0, 0, 0, '2020-01-01'),
+              -- not concepts to be suppressed --
+              (12, 112, 0, 0, 0, 1111111, 0, 0, 0, '2020-01-01'),
+              (13, 113, 0, 0, 0, 2222222, 0, 0, 0, '2020-01-01')
+            """)
+
+        insert_observation_query = observation_data_template.render(
+            project_id=self.project_id, dataset_id=self.dataset_id)
+
+        # Load test data
+        self.load_test_data([f'''{insert_observation_query};'''])
+
+    def test_registered_cope_survey_suppression(self):
+        # Expected results list
+        tables_and_counts = [{
+            'fq_table_name':
+                f'{self.project_id}.{self.dataset_id}.observation',
+            'fq_sandbox_table_name':
+                f'{self.project_id}.{self.sandbox_id}.'
+                f'{self.rule_instance.sandbox_table_for("observation")}',
+            'loaded_ids': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13],
+            'sandboxed_ids': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11],
+            'fields': [
+                'observation_id', 'person_id', 'observation_concept_id',
+                'observation_type_concept_id', 'value_as_concept_id',
+                'observation_source_concept_id', 'value_source_concept_id',
+                'qualifier_concept_id', 'unit_concept_id'
+            ],
+            'cleaned_values': [(12, 112, 0, 0, 0, 1111111, 0, 0, 0),
+                               (13, 113, 0, 0, 0, 2222222, 0, 0, 0)]
```

Comment:
If testing the null record _I think_ this would become:
```suggestion
                               (13, 113, 0, 0, 0, 2222222, 0, 0, 0),
                               (14, 114, 0, 0, 0, null, 0, 0, 0)]
```